proto/rill/admin/v1/api.proto

```diff
@@ -1839,14 +1840,19 @@ message GetReportMetaRequest {
   string project_id = 1;
   string branch = 2;
   string report = 3;
-  map<string, string> annotations = 4;
+  rill.runtime.v1.ReportSpec spec = 6;
+  map<string, string> annotations = 4 [deprecated = true];
```

Comment:
Can this just be removed now? And in the handler just exit early without external user URLs if `spec` is not set. Then the backwards compatibility should be there (and anyway it only matters for the few seconds during release rollout where the runtime is on an older version).